src/index.js

```diff
@@ -1,7 +1,349 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
+import React from "react";
+import ReactDOM from "react-dom";
+import VTMap from "./Components/VTMap.js";
+import Button from "./Components/Buttons.js";
+import Display from "./Components/InfoBox.js";
+import borderData from "./Components/border.js";
+import leafletPip from "leaflet-pip";
+import L from "leaflet";
+import GuessBox from "./Components/guessBox.js"
+import "./CSS/index.css" 
+import Move from "./Components/Move.js"
+import GameWon from "./Components/gameWon.js"
 
-ReactDOM.render(
-  <h1>Hello World!</h1>,
-  document.getElementById('root')
-)
\ No newline at end of file
+let localStore = window.localStorage;
+
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+
+    this.state = {
+      initialLat: 43.9,
+      initialLng: -72.7317,
+      currentLat: 43.9,
+      currentLng: -72.7317,
+      town: "?",
+      county: "?",
+      latDisplay: "?",
+      lngDisplay: "?",
+      startDisabled: false,
+      guessDisabled: true,
+      quitDisabled: true,
```

Comment:
Using what i mentioned above, you can combine these into one item in state rather then 3

Reply:
Fixed!